src/views/passwordReset.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
```

Comment:
Hello @IssaIan ,
Please follow this convention to name your containers. 
`${NameOfContainer}Container` eg: `ArticleListContainer.jsx`
It is what the team agreed on.